vermin/arguments.py

```diff
@@ -304,6 +325,21 @@ def parse(self, config, detect_folder=None):
       elif arg == "--no-exclude":
         config.clear_exclusions()
         path_pos += 1
+      elif arg == "--exclude-regex":
+        if (i + 1) >= len(self.__args):
+          print(r"Exclusion requires a regex! Example: --exclude-regex '.+\.pyi'")
```

Comment:
Done! I have also changed the example here to use `'\\.pyi$'`. When I wrote that print statement I was still using `re.match()` instead of `re.search()`, but with the `re.search()` version we can avoid having to type out the `.+` prefix.